lib/formatter/csv/classification.rb

```diff
@@ -51,6 +51,10 @@ def subject_data
         end.to_json
       end
 
+      def subject_ids
+          classification.subject_ids.map {|id| cache.subject(id).id }.to_json
```

Comment:
We needed the cache for subjects because `classification.subjects` didn't use to be an actual association (the list of subject_ids was stored as a serialized column). So the cache then loaded all the subjects for a batch of classifications (implementing `find_each` behaviour for this non-association). 

I've since turned that into a normal `has_many :subjects, through: :classification_subjects`, so we could probably just use Rails' eager loading for this now.